torch/_inductor/template_heuristics/triton.py

```diff
@@ -653,6 +653,8 @@ def _scale_mm_configs(
 
     def _get_exceeding_shared_memory_checker(
         self,
+        has_sm_layout_conversion: bool = False,
+        layout_conversion_byte_size: int = 0,
```

Comment:
I like conceptually separating these, but I can't help but notice all the math works out if 0 maps to False.